docs/docs/adding-search.md

```diff
@@ -1,68 +1,70 @@
 ---
-title: Adding Search
-overview: true
+शीर्षक: खोज जोड़ना
```

Comment:
`शीर्षक` => `title`

do not translate this, this will be used by to read title from the file